app/controllers/requests.rb

```diff
@@ -1,15 +1,21 @@
 class MakersBnb < Sinatra::Base
   post '/request/new' do
-    user = current_user
+    space = Space.get(params[:space_id])
+    booking_from = space.bookings.map(&:from_date)
+    booking_to = space.bookings.map(&:end_date)
+    request_range=(params[:start_date]..params[:end_date])
+    check_booking_from_date(booking_from, request_range)
+    check_booking_to_date(booking_to, request_range)
```

Comment:
If consolidating `#check_booking_from` and `#check_booking_to` (and refactoring the resulting method), would it be logical to do a single method call, passing in the three arguments (e.g., `check_booking_validity(booking_from, booking_to, request_range)`)?